logger.go

```diff
@@ -121,6 +129,22 @@ func (l *Logger) output(s severity, txt string) {
 	}
 }
 
+// Close closes all the underlying log destinations, which will flush all the
```

Comment:
I don't think this is needed as the caller is in control of the log file, see the README. Having the logger close the file on the Fatal call does make sense as the defer function wont happen on os.Exit.

Reply:
Shouldn't we close syslog conn and unregister windows event log handler?  

> The Init function returns a logger so you can setup multiple instances if you wish, only the first call to Init will set the default logger:

In the case mentioned in the README where someone creates a Logger in a function scope, and there should be a way to cleanly shut it down upon returning.

Reply:
Ah, I din't think about creating a logger at a functions scope, yeah then having a close method makes sense. Can you update the readme (defer loggerOne.Close())

Reply:
Done